html/web/themes/custom/base_starterkit/components/molecules/paragraphs/b_text/b_text.twig

```diff
@@ -1 +1,3 @@
-{{ content }}
+<div class="text-render main-container flex flex-col gap-4 lg:gap-6">
+  {{ content }}
+</div>
```

Comment:
Please remember to add newline at the end of file. We don't like this red circle with dash in PR 😛